frontends/web/src/routes/device/deviceswitch.tsx

```diff
@@ -24,6 +24,7 @@ export interface Devices {
 }
 
 interface Props {
+    key?: any;
```

Comment:
@KasparEtter @benma  -- after updating preact to the latest, TS has been complaining that it cannot assign `devices` to the property `key` (occurs in `app.tsx`), do you know a better work around for this than defining it as `any`?

Reply:
Can you post the exact error message here? Or otherwise tell me what I need to look for resp. change if I check out your branch?

Reply:
First error I get is
```
✖ ERROR [at-loader] ./src/app.tsx:181:29
    TS2326: Types of property 'key' are incompatible.
  Type 'Devices' is not assignable to type 'string | number | undefined'.
    Type 'Devices' is not assignable to type 'number'.
✖ ERROR [at-loader] ./src/app.tsx:186:29
    TS2326: Types of property 'key' are incompatible.
  Type 'Devices' is not assignable to type 'string | number | undefined'.
    Type 'Devices' is not assignable to type 'number'.
```

and when I try to define key to be `Devices`, which is
```
interface Devices {
    [deviceID: string]: string;
}
```
I get
```
✖ ERROR [at-loader] ./src/app.tsx:181:29
    TS2326: Types of property 'key' are incompatible.
  Type 'Devices' is not assignable to type '(Devices & string) | (Devices & number) | undefined'.
    Type 'Devices' is not assignable to type 'Devices & number'.
      Type 'Devices' is not assignable to type 'number'.
✖ ERROR [at-loader] ./src/app.tsx:186:29
    TS2326: Types of property 'key' are incompatible.
  Type 'Devices' is not assignable to type '(Devices & string) | (Devices & number) | undefined'.
    Type 'Devices' is not assignable to type 'Devices & number'.
      Type 'Devices' is not assignable to type 'number'.
```